src/python/pants/backend/docker/target_types.py

```diff
@@ -367,6 +368,28 @@ class DockerImageBuildImageCacheFromField(
     docker_build_option = "--cache-from"
 
 
+class DockerImageBuildImageOutputField(
+    DockerBuildOptionFieldMultiValueDictMixin, DictStringToStringField
+):
```

Comment:
How about adding a `DockerBuildKitOptionField` to the mix?
That will make the `issubclass(..)` cleaner as well when we collect the build options.

```suggestion
class DockerBuildKitOptionField:
    """Mixin to indicate a BuildKit specific option."""
    
class DockerImageBuildImageOutputField(
    DockerBuildKitOptionField,
    DockerBuildOptionFieldMultiValueDictMixin,
    DictStringToStringField,
):
```

Reply:
I'm on the fence regarding the name `DockerBuildxOptionField` vs `DockerBuildKitOptionField`. I find the latter more readable..